nucliadb_utils/nucliadb_utils/storages/storage.py

```diff
@@ -53,6 +54,8 @@
 DEADLETTER = "deadletter/{partition}/{seqid}/{seq}"
 OLD_INDEXING_KEY = "index/{node}/{shard}/{txid}"
 INDEXING_KEY = "index/{kb}/{shard}/{resource}/{txid}"
+# temporary storage for large stream data
+MESSAGE_KEY = "message/{kbid}/{mid}"
```

Comment:
nit: maybe it would be better if the key contained the resource uuid? Just for easier introspection / debugging